schemas/rule.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used to generate alerts or other events.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that generate alerts or other events.
+   Examples of data sources that would populate the rule fields include: network admission control platforms, network or 
+   host IDS/IPS, network firewalls, web access firewalls, url filters, endpoint detection and response (EDR) systems, etc.  
+  
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Rule ID
+      description: >
+        A rule ID that is unique within the scope of an agent, observer, or other entity using the rule for detection of this event.
+
+      example: 101 
+
+    - name: uuid
+      level: extended
+      type: keyword
+      short: Rule UUID
+      description: >
+        A rule ID that is unique within the scope of a set or group of agents, observers, or other entities using the rule for detection of this event.
+
+      example: 1100110011
+
+    - name: version
+      level: extended
+      type: keyword
+      short: Rule Version
+      description: The version / revision of the rule being used for analysis.
+      example: 1.1
+
+    - name: name
+      level: extended
+      type: keyword
+      short: Rule Name
+      description: The name of the rule or signature generating the event.
+      example: Filter_DOH
+
+
+    - name: description
+      level: extended
+      type: keyword
+      short: Rule Description
+      description: The description of the rule generating the event.
+      example: Block requests to public DNS over HTTPS / TLS protocols
+
+    - name: type
+      level: extended
+      type: keyword
+      short: Rule Type
+      description: >
+         A categorization value keyword used by the entity using the rule for detection of this event.
+
+      example: DNS
+      
+    - name: ruleset
+      level: extended
+      type: keyword
+      short: Rule Ruleset
+      description: >
+         Name of the ruleset, policy, group, or parent category in which the rule used to generate this event is a member.
+
+      example: Standard_Protocol_Filters
+
+    - name: reference
+      level: extended
+      type: keyword
+      short: Rule Reference URL
+      description: >
+        Text URL pointing to additional information about the rule used to generate this event.  
+        For example a link to the platform used to manage the observers, a link to the vendors documentation regarding the rule, 
+        or a link to more information on the specific action that triggered the rule.
```

Comment:
I'd like to rephrase this section a bit, and split in two paragraphs. Here's my take.

```
      description: >
        Reference URL to additional information about the rule used to generate this event.

        The URL can point to the vendor's documentation about the rule.
        If that's not available, it can also be a link to a more general page describing this type of alert.
```

You'll note that I'm dropping the part about linking directly to another platform. If I understand correctly, this was meant to link directly in a system, to the page about this specific instance the alert.

While this is a very relevant piece of information, I'd like to capture such specific links in a different field. I'm trying to establish a pattern where the `.reference` URL fields are meant to be documentation-style references about a thing.

I'm still looking for a good name for the field meant to capture deep links to another system. Suggestions welcome :-)

Reply:
sounds good to me